pkg/services/publicdashboards/service/utils.go

```diff
@@ -0,0 +1,22 @@
+package service
+
+import (
+	"encoding/json"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/services/publicdashboards/models"
+)
+
+func UnmarshalDashboardAnnotations(sj *simplejson.Json) (*models.AnnotationsDto, error) {
```

Comment:
We're creating a "junk drawer" by naming this utils.go. Let's rename this file "annotations.go"